docs/book/standard-filters.md

```diff
@@ -234,12 +238,12 @@ To set types after instantiation, use the `setType()` method.
 
 ### Localized Booleans
 
-As mentioned previously, `Zend\Filter\Boolean` can also recognise localized "yes" and "no" strings.
+As mentioned previously, `Zend\Filter\Boolean` can also recognize localized "yes" and "no" strings.
```

Comment:
Is there a reason why the spelling was changed here? So far we have no definition for a standard dialect in the documentation.

Reply:
I gave it some thought and research when making the change. I decided to change it for 2 reasons: There doesn't seem to be any other indication that it is written in a specific dialect -- e.g., no other usage of `ise` vs `ize`. So the first reason is for consistency. The 2nd reason is because I ran across [this](https://www.google.com/search?rlz=1C5CHFA_enUS756US756&ei=lBBLXLy8GuvHjgSr4ZeoDQ&q=recognize+or+recognise&oq=recognize+vs+recogn&gs_l=psy-ab.3.0.0i10l6.104645.109359..110818...1.0..0.110.1848.17j3....2..0....1..gws-wiz.......0i71j35i39j0i131j0j0i67j0i22i10i30.fOLrJ7Ih3Hk) while researching it. I figured the `ize` variant was "better" globally. I'm no expert in linguistics but, there you go.

Reply:
@marcguyer 
Thank you for your detailed answer. In the past I have done the same research several times.
The conclusion is we have no definition for a standard dialect. Therefore, I see no need to change anything in this direction and we should do this kind of discussion in the [documentation issue tracker](https://github.com/zendframework/documentation/issues).

Thanks again for your effort!